wrappers/nodejs/test/test-functional.js

```diff
@@ -234,65 +262,58 @@ describe('Colorizer test', function() {
   });
 });
 
+if (!(isRecord || isPlayback)) {
+  describe('Pointcloud and Points test', function() {
+    let pipe;
+    let frameset;
+    let color;
+    let depth;
+    let pc;
+    let ctx;
+
+    before(function() {
+      ctx = new rs2.Context();
+      pc = new rs2.PointCloud();
+      pipe = new rs2.Pipeline(ctx);
+      pipe.start();
+      frameset = pipe.waitForFrames();
+      color = frameset.colorFrame;
+      depth = frameset.depthFrame;
+    });
 
-describe('Pointcloud and Points test', function() {
-  let pipe;
-  let frameset;
-  let color;
-  let depth;
-  let pc;
-  let ctx;
-
-  before(function() {
-    ctx = new rs2.Context();
-    pc = new rs2.PointCloud();
-    pipe = new rs2.Pipeline(ctx);
-    pipe.start();
-    frameset = pipe.waitForFrames();
-    color = frameset.colorFrame;
-    depth = frameset.depthFrame;
-  });
-
-  after(function() {
-    pipe.stop();
-    pipe.destroy();
-    pc.destroy();
-    ctx.destroy();
-    pc = undefined;
-    ctx = undefined;
-    pipe = undefined;
-    frameset = undefined;
-    color = undefined;
-    depth = undefined;
-  });
+    after(function() {
+      pipe.stop();
+      rs2.cleanup();
+    });
 
-  it('map and calculate test', () => {
-    assert.equal(pc instanceof rs2.PointCloud, true);
+    it('map and calculate test', () => {
+      assert.equal(pc instanceof rs2.PointCloud, true);
 
-    pc.mapTo(color);
-    const points = pc.calculate(depth);
-    const cnt = depth.width*depth.height;
-    assert.equal(points instanceof rs2.Points, true);
-    assert.equal(points.size, cnt);
-    const texCoordinates = points.textureCoordinates;
+      pc.mapTo(color);
+      const points = pc.calculate(depth);
+      const cnt = depth.width * depth.height;
+      assert.equal(points instanceof rs2.Points, true);
+      assert.equal(points.size, cnt);
+      const texCoordinates = points.textureCoordinates;
 
-    assert.equal(points.vertices instanceof Float32Array, true);
-    assert.equal(texCoordinates instanceof Int32Array, true);
+      assert.equal(points.vertices instanceof Float32Array, true);
+      assert.equal(texCoordinates instanceof Int32Array, true);
```

Comment:
indent

Reply:
@halton The indent was added because a new if condition clause was added to wrap these code. The if condition was to check whether the test is under playback mode, This is to avoid running the test under playback mode in travis, and we can still run the test when triggered manually.

Reply:
Oh, I missed that if block, so forget this one.